src/rdpc/query/fetchAnalysesWithSpecimens.ts

```diff
@@ -119,4 +102,11 @@ const fetchAnalysesWithSpecimens = async ({
   });
 };
 
+const retryConfig = {
```

Comment:
decreased retry times to 3, retry delay is 1000 which is smaller than fetch aligment/sanger/mutect queries' retry delay, because this query takes shorter time to complete.

Reply:
retry time should be based on the needs of the service we are calling, not the duration of the call itself. That said, I don't know what the correct timing is. If this is helping then let's use this for now.

Reply:
👍